scripts/github/generate_release_text.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'changelogerator'
+require 'git'
+require 'erb'
+require 'toml'
+
+version = ENV['GITHUB_REF']
+token = ENV['GITHUB_TOKEN']
+
+polkadot_path = ENV['GITHUB_WORKSPACE'] + '/polkadot/'
+pg = Git.open(polkadot_path)
+
+# Generate an ERB renderer based on the template .erb file
+renderer = ERB.new(
+  File.read(ENV['GITHUB_WORKSPACE'] + '/polkadot/scripts/github/polkadot_release.erb'),
+  trim_mode: '<>'
+)
+
+# get last polkadot version. Use handy Gem::Version for sorting by version
+last_version = pg
+              .tags
+              .map(&:name)
+              .grep(/^v\d+\.\d+\.\d+.*$/)
+              .sort_by { |v| Gem::Version.new(v.slice(1...)) }[-2]
+
+polkadot_cl = Changelog.new(
+  's3krit/polkadot', version, last_version, token: token
```

Comment:
I think there's a typo here?
```suggestion
  'paritytech/polkadot', version, last_version, token: token
```

Reply:
Good catch :) result of me testing on a fork